src/coreclr/jit/emit.cpp

```diff
@@ -3864,6 +3869,21 @@ void emitter::emitDispGCinfo()
     printf("\n\n");
 }
 
+void emitter::emitDispJumpList()
```

Comment:
Needs function header comment

Reply:
Done. It's worth noting that many of the other Disp functions around it don't have headers.